docs/book/src/topics/flatcar.md

```diff
@@ -0,0 +1,27 @@
+# Use Flatcar images
+
+[Flatcar](https://flatcar.org) is a Linux based OS designed to run containers.
```

Comment:
To make this new section available in the book, it needs to also be added to the list of topics in `./docs/book/src/SUMMARY.md` [here](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/main/docs/book/src/SUMMARY.md), maybe as a new sub-section under `Self-managed clusters`?